modules/channel-web/src/views/web/index.jsx

```diff
@@ -465,6 +465,10 @@ export default class Web extends React.Component {
     const url = `/mod/channel-web/messages/${userId}`
     const config = { params: { conversationId: this.state.currentConversationId }, ...this.axiosConfig }
 
+    if (data.type === 'visit') {
+      data.timezone = moment().utcOffset() / 60
+    }
+
```

Comment:
Also, this logic could be added directly in the method which sends the user visit:
```this.handleSendData({
type: 'visit',
text: 'User visit',
timezon: moment().utcOffset() / 60
```

Reply:
yes, good idea